lib/litani_report.py

```diff
@@ -310,10 +310,12 @@ def render(run, report_dir):
 
     svgs = render_runtimes(run, env, report_dir)
 
+    litani_zip_path = os.environ.get("LITANI_ZIP_PATH")
```

Comment:
This is a bit generic, it'll get confusing if we have other zip files in the future (which I expect we will). Can I suggest `LITANI_REPORT_ARCHIVE_PATH`?

Reply:
Also `os.getenv` is shorter than `os.environ.get`